src/main/java/frc/robot/RobotContainer.java

```diff
@@ -27,13 +34,24 @@ public class RobotContainer {
 
     public RobotContainer() {
 
+        // Initialize subsystems
+        // TODO: Placeholder PID values
+        leftMotors = new MotorSubsystem("Left Motors", new PID(1, 0, 0), TalonFXInvertType.Clockwise,
+                new MetersPerSecond(0.01), new FeedForwardGains(0.18157, 2.3447, 0.54597),
```

Comment:
You can extract the Meters Per Second and Feed Forward Gains into a variable and reuse that variable for both motor instances since they're the same value.

You can also do the same for PIDs if you think they'll be the same between both motors.

- [ ] Extract Meters Per Second into a variable
- [ ] Extract Feed Forward Gains into a variable.
- [ ] Use those variables to initialize both motor instances